pkg/manifestclient/list.go

```diff
@@ -74,6 +75,13 @@ func (mrt *manifestRoundTripper) listAll(requestInfo *apirequest.RequestInfo) ([
 		if retList.GroupVersionKind() != kind.listKind {
 			return nil, fmt.Errorf("inconsistent list kind: got %v, expected %v", retList.GroupVersionKind(), kind.listKind)
 		}
+		if requestInfo.LabelSelector != "" {
```

Comment:
nit: by checking the value of the selector inside the `filterByLabelSelector` we would get:
```
retList, err = filterByLabelSelector(retList, requestInfo.LabelSelector)
if err != nil {
 return err
}
```

Reply:
Done